network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1142,62 +1139,62 @@ def about_page(self, request):
         context = self.get_context(request)
         context['pagetype'] = 'about'
         context['pageTitle'] = pgettext(
-            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
-            '*privacy not included') + ' - How to Use This Guide'
+            '*privacy not included can be localized.',
+            'How to use *privacy not included')
         return render(request, "about/how_to_use.html", context)
 
     @route(r'^about/why/$', name='about-why-view')
     def about_why_page(self, request):
         context = self.get_context(request)
         context['pagetype'] = 'about'
         context['pageTitle'] = pgettext(
-            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
-            '*privacy not included') + ' - Why We Made This Guide'
+            '*privacy not included can be localized.',
+            'Why we made *privacy not included')
         return render(request, "about/why_we_made.html", context)
 
     @route(r'^about/press/$', name='press-view')
     def about_press_page(self, request):
         context = self.get_context(request)
         context['pagetype'] = 'about'
-        context['pageTitle'] = pgettext(
+        context['pageTitle'] = gettext('Press') + ' | ' + pgettext(
```

Comment:
Just adding one more comment, to specify it’s that kind of press, not a verb/action.

```suggestion
        context['pageTitle'] = pgettext('Noun, media', 'Press') + ' | ' + pgettext(
```

Reply:
Thanks for making this more clear! @TheoChevalier